docs/USAGE.md

```diff
@@ -5,50 +5,22 @@ This document describes various core Plutarch usage concepts.
 <details>
 <summary> Table of Contents </summary>
 
-- [Applying functions](#applying-functions)
 - [Conditionals](#conditionals)
 - [Recursion](#recursion)
-- [Do syntax with `TermCont`](#do-syntax-with-termcont)
-- [Do syntax with `QualifiedDo` and `Plutarch.Monadic`](#do-syntax-with-qualifieddo-and-plutarchmonadic)
-- [Deriving typeclasses for `newtype`s](#deriving-typeclasses-for-newtypes)
+- [Do syntax with `TermCont`](#do-syntax-with--termcont-)
+- [Do syntax with `QualifiedDo` and `Plutarch.Monadic`](#do-syntax-with--qualifieddo--and--plutarchmonadic-)
+- [Deriving typeclasses for `newtype`s](#deriving-typeclasses-for--newtype-s)
 - [Deriving typeclasses with generics](#deriving-typeclasses-with-generics)
+- [`plet` to avoid work duplication](#-plet--to-avoid-work-duplication)
+- [Tracing](#tracing)
+- [Raising errors](#raising-errors)
+- [Unsafe functions](#unsafe-functions)
 
-</details>
-
-# Applying functions
-
-You can apply Plutarch level functions using `#` and `#$` (or `papp`). Notice the associativity and precedence of those operators:
-
-```haskell
-infixl 8 #
-
-infixr 0 #$
-```
-
-`#$` is pretty much just `$` for Plutarch functions. But `#` is left associative and has a high precedence. This essentially means that the following:
-
-```haskell
-f # 1 # 2
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
```

Comment:
what's this?

Reply:
Ah -- hyperlink from the TOC generator I used.

BTW, it appears that it had some different anchor rewriting rules than GitHub uses. There were a bunch of broken links across the various files that I touched, but they should all be fixed now.